tests/failpoints/cases/test_snap.rs

```diff
@@ -545,24 +545,33 @@ fn test_sending_fail_with_net_error() {
     let mut cluster = new_server_cluster(1, 2);
     configure_for_snapshot(&mut cluster);
 
-    let on_send_store_fp = "receiving_snapshot_net_error";
-
     let pd_client = Arc::clone(&cluster.pd_client);
     // Disable default max peer count check.
     pd_client.disable_default_operator();
     cluster.run_conf_change();
 
     cluster.must_put(b"k1", b"v1");
+    let ready_notify = Arc::default();
+    let (notify_tx, notify_rx) = mpsc::channel();
+    cluster.sim.write().unwrap().add_send_filter(
+        1,
+        Box::new(MessageTypeNotifier::new(
```

Comment:
I think you can use custom filter, that pass the first `MsgSnapshot` message, then drop all `MsgSnapshot` latter.

Reply:
yes